metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java

```diff
@@ -278,7 +278,7 @@ private void requireValidCode(final char code, final char[] validCodes) {
                     return;
                 }
             }
-            throw new FormatException("invalid code '" + code + "'; allowed codes are: " + Arrays.toString(validCodes));
+            throw new FormatException("invalid code in leader'" + code + "'; allowed codes are: " + Arrays.toString(validCodes));
```

Comment:
@dr0i is it possible to add the info about the leader element? like this:
```suggestion
            throw new FormatException("invalid code in leader literal literal'" + name + code + "'; allowed codes are: " + Arrays.toString(validCodes));
```

Reply:
You've proposed to add  `literal literal` as message. And a variable `name` which is not declared in the method nor globally (meaning: you cannot use it here). 
What you probably meant to output was something like e.g. `TYPE_OF_CONTROL_CODES` or `ENCODING_LEVEL_CODES` so that a librarian knows what leader position is wrong? We could do this, but have to pass this information somehow into the method via a parameter.